zk_toolbox/crates/config/src/explorer.rs

```diff
@@ -0,0 +1,111 @@
+use std::path::{Path, PathBuf};
+
+use serde::{Deserialize, Serialize};
+use xshell::Shell;
+
+use crate::{
+    consts::{
+        EXPLORER_CHAIN_CONFIG_FILE, EXPLORER_RUNTIME_CONFIG_FILE, LOCAL_APPS_PATH,
+        LOCAL_CHAINS_PATH, LOCAL_CONFIGS_PATH, LOCAL_GENERATED_PATH,
+    },
+    traits::{ReadConfig, SaveConfig, ZkToolboxConfig},
+};
+
+/// Explorer configuration file. This file is auto-generated during the "explorer" command
+/// and is used to inject the runtime configuration into the explorer app.
+#[derive(Serialize, Deserialize, Debug, Clone)]
+#[serde(rename_all = "camelCase")]
```

Comment:
Why do we want camelCase? Does it have to do with `js`?

Reply:
Yes, this is for frontend - javascript file.